### PR TITLE
Welcome to my capstone project

### PR DESCRIPTION
**Pull request by mariegrace31**
In this milestone I :
- Created 2 HTML pages, the home page, and the About page
- Used Flexbox to place elements on the page.
- Used images and backgrounds to enhance the look of a website.
- Used JavaScript to manipulate DOM elements.
- Used objects to store and access data.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,89 @@
...
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need: to dowload it or clone it to your machine.
+### Setup
+Clone this repository to your desired folder: https://github.com/mariegrace31/capstone_one.git
+### Install
+Install this project with: cd my-project npm install
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (usorfaitheloho) [inline README.md]: You did a great job in implementing your capstone project, please add a live link showing your deployed project using github pages to enable a visitor have access to your live page from your README file 😄

File: README.md [hunk 2]
```diff
@@ -1 +1,89 @@
...
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for giving me this opportunity and you too for the time you took going through this project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- LICENSE -->
```
Comment (usorfaitheloho) [inline README.md]: Kindly give appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/) in the acknowledgement section of your README file 😄

File: home.html [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+        <p class="explication">Use our nail products to ensure that your nails are in good condition and the skin on hands and feet is well cared.</p>
+      </div>
+      <div class="skincare">
+        <img class="product-image" src="images/hair.png"> 
+        <h3>Hair</h3>
+        <p class="explication">Our hair products will strengthen your hair follicles and enhance hair growth.</p>
+      </div>
+    </div>
+    <a class="button">View more products</a>
+  </section>
```
Comment (usorfaitheloho) [inline home.html]: Kindly implement the `hover` feature for the project section so you have a better UI

Design given :
image

your design :
image

File: home.html [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+  <div class="artists" id="Artists">
+    <h2 class="arts">Our Beautician Artists</h2>
+    <hr class="title-underline">
+    <div class="artist-wrapper"></div>
+    <!--
```
Comment (usorfaitheloho) [inline home.html]: Kindly implement the bottom section articles  to be  organized in two columns of three articles eachas seen in the deign and also add the top background image as it is the design given.

Your design :
image

Figma design :
image

File: home.html [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+    </div>
+  </div>
+  <div class="Footer">
+    <img class="LOGO2"src="images/our logo.png">
+    <p class="footer2">@Flawless <br> Some Rights Reserved.</p>
+  </div>
+
+  <script src="js/script.js"></script>
+</body>
+</html>
```
Comment (usorfaitheloho) [inline home.html:112]: please kindly implement these changes for your mobile version of the home page :

- [ ] Please increase the margin before the `view more products` button
- [ ] kindly implement the `more` button to display more makeup artist as it is in the design 😄

File: home.html [hunk 6]
```diff
@@ -0,0 +1,132 @@
...
+        <ul>
+         <li><i class="fa fa-twitter"></i></li> 
+         <li><i class="fa fa-instagram"></i></li> 
+         <li><a href="">French</a></li>
+         <li><a href="">My page</a></li>
+         <li><a href="">Logout</a></li>
+        </ul>
+      </div>
+      <div class="nav-bar">
+        <a href="home.html"><img class="click" src="images/logo.png"></a>
```
Comment (usorfaitheloho) [inline home.html]: please kindly add `alt ` attribute for all your image property as it shows good html semantics 😄

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
> Hi @mariegrace31  ,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

  [ ](https://camo.githubusercontent.com/679757134a8fee76cfe1941f9f3ecb0e0cf56dd02da4498268ce31f0dacdfca3/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)
> 
> ## Highlights
> * All linter checks passed ✔️
> * Descriptive pull request ✔️
> * Used GitHub flow ✔️
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (mariegrace31) [reply README.md]: @usorfaitheloho Thanks for your review. I put the live link of my project, under the authors

File: home.html [hunk 7]
```diff
@@ -0,0 +1,112 @@
...
+        <p class="explication">Our hair products will strengthen your hair follicles and enhance hair growth.</p>
+      </div>
+    </div>
+    <a class="button">View more products</a>
+  </section>
+  <div class="artists" id="Artists">
+    <h2 class="arts">Our Beautician Artists</h2>
+    <hr class="title-underline">
+    <div class="artist-wrapper"></div>
+    </div>
```
Comment (Tufahel) [inline home.html:91]: ***[Figma Design]***
image

- [ ] You can see that there are background designs for every `images` in this section in the given design. As per the previous reviewer's suggestion kindly fix this to match the given design.

File: README.md [hunk 8]
```diff
@@ -1 +1,89 @@
...
+Vs Code using HTML, CSS and JavaScript.
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+- [Feature_1]: Setup a professional Github repository
+- [Feature_2]: Setup microverse linters
+- [Feature_3]:Use images and backgrounds to enhance the look of the website.
+- [Feature_4]: Use JavaScript to manipulate DOM elements
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```
Comment (Tufahel) [inline README.md:35]: - [ ] As per the previous reviewer's suggestion kindly add a section named `Live Demo` and add a link in this section. As for a professional README file a `Live Link` is must required. Kindly fix this issue.
- image

**Review by Tufahel — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @mariegrace31 :wave:  

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 🌟

Alt Text

✔️ Descriptive pull request.
✔️ Linter checks are passing.
✔️ Follow the correct GitHub flow.
✔️ Commit messages are meaningful.

## Required Changes ♻️

***Please kindly check the comments/review below.***

## Optional suggestions 🤷‍♂️

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:✔️✔️ 

Hi @mariegrace31   ✔️✔️ ✔️✔️ 

Your project is complete! Great job for taking time to make the changes🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request✔️✔️ 
- Behance design well followed ✔️✔️ 
- well deployed✔️✔️ 
- work well presented✔️✔️ 
- No linter errors✔️✔️ 


---[optional] Please kindly add the future feature section with some content of what you might wish to add in your project later on. this can help as a reminder to you. 
<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

smile

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**